cognite_toolkit/_cdf_tk/templates/data_classes/_config_yaml.py

```diff
@@ -178,7 +178,7 @@ def get_selected_modules(
         if verbose:
             print("  [bold green]INFO:[/] Selected modules:")
             for module in selected_modules:
-                print(f"    {module}")
+                print(f"    {MODULE_PATH_SEP.join(module)!s}")
```

Comment:
Here you need to be careful as module can also be a `str` which would fail with this code.